runtime/doc/lua.txt

```diff
@@ -4308,4 +4308,29 @@ vim.text.hexencode({str})                               *vim.text.hexencode()*
     Return: ~
         (`string`) Hex encoded string
 
+
+==============================================================================
+Lua module: vim.tohtml                                            *vim.tohtml*
+
+tohtml({winnr}, {outfile}, {opt})                   *require"tohtml".tohtml()*
```

Comment:
Wouldn't it make more sense for this to return a string? Then the user can write it to a file if they wish.

Reply:
That should then be another function, as this function is designed to be the same as `:TOhtml`.
But I don't know what that function should be called...